nearcore/src/lib.rs

```diff
@@ -232,6 +233,9 @@ pub fn start_with_config_and_synchronization(
         config.client_config.log_summary_period,
     )?;
 
+    // Initialize genesis STATE either from config or state/storage before other components.
```

Comment:
nit: rather than capitalizing the state (which can be a bit misleading, maybe the capitalized "STATE" refers to something specific), maybe elaborate a bit more about the difference between genesis and genesis state.

Reply:
updated comments